src/reactviews/pages/SchemaCompare/components/SchemaDifferences.tsx

```diff
@@ -3,35 +3,88 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { useContext } from "react";
+import * as React from "react";
+import { FixedSizeList as List, ListChildComponentProps } from "react-window";
 import {
-    Checkbox,
-    createTableColumn,
-    Table,
+    useScrollbarWidth,
+    useFluent,
     TableBody,
     TableCell,
-    TableColumnDefinition,
+    TableRow,
+    Table,
     TableHeader,
     TableHeaderCell,
-    TableRow,
+    createTableColumn,
+    useTableFeatures,
+    useTableSelection,
+    TableRowData as RowStateBase,
+    TableColumnDefinition,
+    Checkbox,
+    makeStyles,
+    Spinner,
 } from "@fluentui/react-components";
-import { FixedSizeList as List } from "react-window";
-import { schemaCompareContext } from "../SchemaCompareStateProvider";
 import { SchemaUpdateAction } from "../../../../sharedInterfaces/schemaCompare";
 import { locConstants as loc } from "../../../common/locConstants";
 import { DiffEntry } from "vscode-mssql";
+import { schemaCompareContext } from "../SchemaCompareStateProvider";
 
-type DiffItem = DiffEntry & {
-    id: number;
-};
+const useStyles = makeStyles({
+    HeaderCellPadding: {
+        padding: "0 8px",
+    },
+
+    PrimaryCheckboxPadding: {
+        paddingLeft: "16px",
```

Comment:
Hmm... there's gotta be a better way to center it than some manual pixel padding on one side

Reply:
The padding is to get the checkbox in the table header to align with the checkboxes that appear for it each diff entry in the table body.